Hardcode not to build docs. Shouldn't be merged upstream; docs are useful!

---
diff --git a/Makefile.am b/Makefile.am
index 7c11310..9bc2e19 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = doc src
+SUBDIRS = src
 ACLOCAL_AMFLAGS = -I m4
 # as per recommendation at <https://www.gnu.org/software/libtool/manual/html_node/Invoking-libtoolize.html>
 
diff --git a/configure.ac b/configure.ac
index b843cf4..750ec17 100644
--- a/configure.ac
+++ b/configure.ac
@@ -134,5 +134,5 @@ AM_ICONV
 
 AC_CHECK_DECLS(O_BINARY, , , [ #include <fcntl.h> ] )
 
-AC_CONFIG_FILES(Makefile doc/Makefile src/Makefile)
+AC_CONFIG_FILES(Makefile src/Makefile)
 AC_OUTPUT
